android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraModule.java

```diff
@@ -654,6 +656,7 @@ private void captureWithOrientation(final ReadableMap options, final Promise pro
         final Boolean shouldMirror = options.hasKey("mirrorImage") && options.getBoolean("mirrorImage");
 
         RCTCamera.getInstance().adjustCameraRotationToDeviceOrientation(options.getInt("type"), deviceOrientation);
+        camera.setPreviewCallback(null);
```

Comment:
This fixes a separate issue where the app was freezing on pressing the capture button. No error was being raised in the android logs, the freezing stopped after I put in this line. 

see: http://stackoverflow.com/a/12427279